src/components/datagrid/data_grid.test.tsx

```diff
@@ -1441,6 +1441,20 @@ Array [
 
   describe('keyboard controls', () => {
     it('supports simple arrow navigation', () => {
+      let pagination = {
```

Comment:
Because of changes to `data_grid.tsx`, this test's pagination values/state needs to be tracked